iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -915,6 +915,10 @@ private void setCommonJobConf(JobConf jobConf) {
     jobConf.set("tez.mrreader.config.update.properties", "hive.io.file.readcolumn.names,hive.io.file.readcolumn.ids");
   }
 
+  public String getTableType(org.apache.hadoop.hive.ql.metadata.Table table) {
```

Comment:
why do you need to pass the table param?

Reply:
Table param is a typo which I didn't notice, I'll remove it. About returning `StorageFormat.StorageHandlerTypes.ICEBERG`, in that case I think we should rename the method `getTableType()` to `getStorageFormatType()`, do you agree?